app/controllers/finance/npq/course_payment_breakdowns_controller.rb

```diff
@@ -26,11 +26,19 @@ def lead_provider_scope
       end
 
       def aggregator
-        ParticipantAggregator.new(
-          statement: @statement,
-          recorder: ParticipantDeclaration::NPQ.where.not(state: %w[voided]),
-          course_identifier: params[:id],
-        )
+        if @statement.past_deadline_date?
+          Finance::NPQ::ParticipantEligibleAndPayableAggregator.new(
+            statement: @statement,
+            recorder: ParticipantDeclaration::NPQ.where.not(state: %w[voided]),
```

Comment:
Wondering what we are currently doing with 'submitted' NPQ declarations, are they to be included too? I know we discussed dealing with these separately at some point but am not clear if that means they should appear on statements

Reply:
I asked about submitted declarations yesterday and they are not to appear on statements but I think this is handled by my PR